config/locales/fr.yml

```diff
@@ -136,34 +136,34 @@ fr:
       title: Changement de mot de passe
       will_email_notice: Nous vous enverrons un email pour changer votre mot de passe.
   multifactor_auths:
-    incorrect_otp:
-    otp_code:
-    require_mfa_disabled:
-    require_mfa_enabled:
+    incorrect_otp: Votre clé OTP est incorrect.
+    otp_code: clé OTP
+    require_mfa_disabled: Votre authentication multifacteur a été activée. Vous devez d'abord la désactiver.
+    require_mfa_enabled: Votre authentication multifacteur n'a pas été activée. Vous devez d'abord l'activer.
     new:
-      title:
-      scan_prompt:
-      otp_prompt:
-      confirm:
-      enable:
-      account:
-      key:
+      title: Activer l'authentication multifacteur
+      scan_prompt: Veuillez scanner le code QR avec votre app d'authentication. Si vous ne pouvez pas scanner le code, veuillez saisir manuellement les informations ci-dessous dans votre app.
```

Comment:
I'm wondering if we should say "QR code", even if it's less french this is the way I heard it most of the time. 🤔

Reply:
I don't know, from Wikipédia: 
![image](https://user-images.githubusercontent.com/22852543/47225169-ba23fb00-d3bd-11e8-9e99-3619e62d9d02.png)

Reply:
I don't know what's correct, but indeed I've never heard "code QR" in real life amongst french people (even non techies). That said I discover that many sites use "code QR" indeed!

Reply:
I am changing it to `QR code` as it is more used this way.